comp/core/workloadmeta/collectors/internal/kubemetadata/kubemetadata.go

```diff
@@ -188,7 +188,7 @@ func (c *collector) parsePods(
 
 	var err error
 	var metadataByNsPods apiv1.NamespacesPodsStringsSet
-	if c.isDCAEnabled() && c.dcaClient.Version().Major >= 1 && c.dcaClient.Version().Minor >= 3 {
+	if c.isDCAEnabled() && c.dcaClient.Version(false).Major >= 1 && c.dcaClient.Version(false).Minor >= 3 {
```

Comment:
I know it was like this before, but should we change this to do a single `c.dcaClient.Version` call?
Depending on the internals of the function it could do 2 separate calls to fetch the version.

Reply:
I would make 2 function calls, and not API calls. 

But in both cases, yes I agree it is better to make only 1 function call 👍